javav2/example_code/glacier/Readme.md

```diff
@@ -22,7 +22,7 @@ For systems with Bash support, once you set the **CLASSPATH**, you can run a par
 
  ## Testing the Amazon Glacier files
 
-You can test the Java code examples for Amazon Glacier by running a test file named **AWSGlacierServiceIntegrationTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+You can test the Java code examples for Amazon Glacier by running a test file named **GlacierTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
```

Comment:
Service naming is 'Amazon Simple Storage Service Glacier' for first mention and 'Amazon S3 Glacier' for subsequent mentions. Several of these need to be updated in the first part of the README.

Reply:
Fixed

Reply:
Fixed